tools/@aws-cdk/prlint/lint.ts

```diff
@@ -265,6 +265,10 @@ export class PullRequestLinter {
       });
     }
 
+    const comments = await this.client.issues.listComments();
+    if (comments.data.find(comment => comment.body?.includes("Exemption Request"))) {
+      body += '\n\nA exemption request has been requested. Please wait for a maintainer\'s review.';
```

Comment:
Super small nit: can we add an emoji or some other eye-catching thing here? otherwise it will be hard to see that the comment has changed.